docs/development_suite/identity-and-access-management/service-account-management.md

```diff
@@ -9,7 +9,7 @@ Mia-Platform Console allows the creation of Service Accounts, which are typicall
 Just like human Users, Service Accounts can be assigned roles at Console, Company, Project or Runtime Environment levels, according to which they can perform different types of actions.
 
 :::note
-To find out more about Roles check out the available [Capabilities](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#users-capabilities-inside-console) that can be assigned to an identity.
+To find out more about Roles check out the available [Capabilities](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#users-capabilities-inside-console) that can be assigned to an [Identity](/development_suite/identity-and-access-management/overview.md#identity-and-access-management).
 :::
 
 ### What can you do with a Service Account?
```

Comment:
Could we insert somewhere here below a link to the Mia-Platform dedicated blog post?